yt/docs/README.md

```diff
@@ -211,3 +437,47 @@ yt check-permission yql write //tmp
     "subject_name" = "yandex";
 }
 ```
+````
+
+------
+
+### Documentation Philosophy
+
+- When writing text, determine your target audience, whom you are writing for. Take into account that YTsaurus users may have different backgrounds and may be from different areas: analysts, engineers, administrators, and developers. The target audience influences the level of detail of your text and the terminology you use.
+- Your text should not disrupt the integrity of the entire document or its narrative logic. When writing content, consider the context of the section where this content is intended to be added.
+- Try to avoid jargon, slang, Anglicisms, and colloquial language. Use commonly accepted terms. Strive to follow the academic style.
+- Accompany your text with examples. Content without examples is perceived as abstract and is hard to assimilate.
+- Use cross-links between the sections.
+- Aim to write concisely and clearly. Do not make sentences too long — up to 15 words per sentence. Split long paragraphs into short ones — up to 5 sentences in one paragraph.
+- If you need any help with content writing, don't hesitate to ask questions in YTsaurus [community chats](#need-help).
+
+------
+
+## How to Build Locally
+
+1. Install [Node.js](https://nodejs.org/ru/download/). The package manager `npm` will be installed simultaneously: it is required for the local build to work.
+2. Install the [@diplodoc/cli](https://www.npmjs.com/package/@doc-tools/docs) package:
+   ```bash
+   $ npm i @diplodoc/cli -g
+   ```
+3. Check that the `yfm` tool is installed correctly. For example, check the version number:
+   ```
+   $ yfm --version
+   ```
+4. Clone YTsaurus repository in case you haven't done it yet:
+   ```bash
+   $ git clone https://github.com/ytsaurus
+   ```
+5. Run the build:
+   ```bash
+   $ yfm -i ytsaurus/yt/docs/ -o ytsaurus-doc-output
+   ```
+6. Navigate to the `ytsaurus-doc-output` folder and then go to the `ru/` or `en/` directory, depending on the documentation language you want to check. When opening the `index.html` file, the main page of the documentation will be shown.
```

Comment:
<img width="1728" alt="Screenshot 2024-03-19 at 14 41 58" src="https://github.com/ytsaurus/ytsaurus/assets/18371266/7d166748-3638-4ec3-b9d9-88892ac749ea">
When opening index.html it looks like this :(
This is very different from the rendering within Yandex and on ytsaurus.tech/docs.
How can we improve this?